### PR TITLE
Add new content element "Custom template"

### LINKED ISSUE #866: Document new content element “Template”
https://github.com/contao/contao/pull/3625

### PR DESCRIPTION
**Pull request by fkaminski**
DE/EN 
~~Hint: The german translation (regarding element name) is not final yet!~~

### CODE REVIEW

File: docs/manual/article-management/content-elements.de.md [hunk 1]
```diff
@@ -1405,3 +1405,16 @@ Das Element generiert folgenden HTML-Code:
...
+
+Das Inhaltselement »Custom template« bietet die Möglichkeit beliebige Templates auszuwählen und dabei 
+eigene, individuelle Platzhalter zu definieren die im jeweiligen Template ausgegeben werden können.
+
+| Info |   |
+| ---- | - |
+| **Template data:** | Angabe eines oder mehrerer Schlüssel/Wert Paare zur Ausgabe im Template. |
+| **Inhaltselement-Template:** | Hier kannst du das Standard-Template `ce_template` überschreiben. |
+
+{{< version "4.13" >}}
```
Comment (fritzmg) [inline docs/manual/article-management/content-elements.de.md]:
```suggestion
{{< version "4.13" >}}

Das Inhaltselement »Custom template« bietet die Möglichkeit beliebige Templates auszuwählen und dabei 
eigene, individuelle Platzhalter zu definieren die im jeweiligen Template ausgegeben werden können.

| Info |   |
| ---- | - |
| **Template data:** | Angabe eines oder mehrerer Schlüssel/Wert Paare zur Ausgabe im Template. |
| **Inhaltselement-Template:** | Hier kannst du das Standard-Template `ce_template` überschreiben. |
```

File: docs/manual/article-management/content-elements.en.md [hunk 2]
```diff
@@ -1240,3 +1240,16 @@ The element generates the following HTML code:
...
+
+The content element »Custom template« offers the possibility to select any template and to define your own, 
+individual placeholders which can be output in the respective template.
+
+| Info |   |
+| ---- | - |
+| **Template data:** | Specification of one or more key/value pairs. |
+| **Content element template:** | Here you can overwrite the default template `ce_template`. |
+
+{{< version "4.13" >}}
```
Comment (fritzmg) [inline docs/manual/article-management/content-elements.en.md]:
```suggestion
{{< version "4.13" >}}

The content element »Custom template« offers the possibility to select any template and to define your own, 
individual placeholders which can be output in the respective template.

| Info |   |
| ---- | - |
| **Template data:** | Specification of one or more key/value pairs. |
| **Content element template:** | Here you can overwrite the default template `ce_template`. |
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/article-management/content-elements.de.md [hunk 3]
```diff
@@ -1407,14 +1407,14 @@ Das Element generiert folgenden HTML-Code:
...
 
 
-## Custom Template {#template}
+## Custom Template
 
 {{< version "4.13" >}}
 
-Das Inhaltselement »Custom template« bietet die Möglichkeit beliebige Templates auszuwählen und dabei 
-eigene, individuelle Platzhalter zu definieren die im jeweiligen Template ausgegeben werden können.
+Das Inhaltselement »Custom template« bietet die Möglichkeit ein Template auszuwählen und dabei individuelle
```
Comment (xchs) [inline docs/manual/article-management/content-elements.de.md]:
```suggestion
Das Inhaltselement »Custom template« bietet die Möglichkeit, ein Template auszuwählen und dabei individuelle 
```

File: docs/manual/article-management/content-elements.de.md [hunk 4]
```diff
@@ -1407,14 +1407,14 @@ Das Element generiert folgenden HTML-Code:
...
 
-## Custom Template {#template}
+## Custom Template
 
 {{< version "4.13" >}}
 
-Das Inhaltselement »Custom template« bietet die Möglichkeit beliebige Templates auszuwählen und dabei 
-eigene, individuelle Platzhalter zu definieren die im jeweiligen Template ausgegeben werden können.
+Das Inhaltselement »Custom template« bietet die Möglichkeit ein Template auszuwählen und dabei individuelle 
+Platzhalter zu definieren die im jeweiligen Template ausgegeben werden können.
```
Comment (xchs) [inline docs/manual/article-management/content-elements.de.md]:
```suggestion
Platzhalter zu definieren, die im jeweiligen Template ausgegeben werden können.
```

**Comment by fkaminski** [comment]
Requires observation: See: [#3950](https://github.com/contao/contao/issues/3950)

**Comment by netzarbeiter** [comment]
I close this pull request because of https://github.com/contao/docs/pull/933.

**Comment by fkaminski** [comment]
@netzarbeiter Please reopen this.
This pull request documents the new content element.
#933 is about the new module.

~~(Even if the final german translation is not sure yet (see [#3950](https://github.com/contao/contao/issues/3950)))~~
Now finally translated ...

**Review by fritzmg — APPROVED** [review]

**Comment by fkaminski** [comment]
Tried to fix conflicts because of #943. Hope this will work ...

**Review by netzarbeiter — APPROVED** [review]
Thank you, Franko

**Pull request merged**